pkg/modules/observability.go

```diff
@@ -90,11 +96,17 @@ const (
 	// OtelCollectorImage - Otel collector image name
 	OtelCollectorImage string = "<OTEL_COLLECTOR_IMAGE>"
 
-	// PscaleObsYamlFile - PowerScale Observability yaml file
-	PscaleObsYamlFile string = "karavi-metrics-powerscale.yaml"
-
 	// OtelCollectorYamlFile - Otel Collector yaml file
 	OtelCollectorYamlFile string = "karavi-otel-collector.yaml"
+
+	// PscaleObsObjectsYamlFile - PowerScale Observability Objects (except deployment) yaml file
+	PscaleObsObjectsYamlFile string = "karavi-metrics-powerscale-objects.yaml"
+
+	// PscaleObsDeploymentYamlFile - PowerScale Observability Deployment yaml file
+	PscaleObsDeploymentYamlFile string = "karavi-metrics-powerscale-deployment.yaml"
+
```

Comment:
Why split 'karavi-metrics-powerscale.yaml' into 2 yaml files?

Reply:
@YianZong , 'karavi-metrics-powerscale.yaml' were parsed to common crclient.Objects, we need to try hard transfer all common obj and find the deplyoment obj, then further transfer the deplyoment obj to DeploymentApplyConfiguration. Then inject container/volume to this DeploymentApplyConfiguration.

To avoid these hard transfer and Marshal/Unmarshal actions and their error handling, the 'karavi-metrics-powerscale.yaml' is divided to "karavi-metrics-powerscale-deployment.yaml" and "karavi-metrics-powerscale-objects.yaml".

Reply:
@YianZong  updated, use one 'karavi-metrics-powerscale.yaml' now